### ISSUE TITLE
GND and VCC markers are reversed？

### ISSUE THREAD

**Issue by CodeMan-P** [creapunk/CLN17#6]
The GND and VCC markings on the right side of the top layer are reversed

![image](https://github.com/creapunk/CLN17/assets/33123544/6871d940-4573-4166-90f5-e7f10e7b1d76)

**Comment by FutureEngineeer** [comment]
@CodeMan-P Indeed. Thanks for noticing) I'll fix it in the documentation for the newer version)
